src/instr/instr_paje_values.cpp

```diff
@@ -15,7 +15,7 @@ value::value(const char* name, const char* color, type_t father)
   if (name == nullptr || father == nullptr){
     THROWF (tracing_error, 0, "can't create a value with a nullptr name (or a nullptr father)");
   }
-  this->ret         = xbt_new0(value, 1);
+  this->ret         = xbt_new1(value, 1); 
```

Comment:
This is where the important thing should take place. Remove this new, and kill the definition of this->ret.